docs/support_table.md

```diff
@@ -0,0 +1,1173 @@
+---
+id: support_table
+title: Support Table
+---
+<!-- {@stylesheet: static/katex.min.css} -->
+<style>
+table tr,
+table td {
+    vertical-align: middle;
+}
+</style>
+
+This is a list of TeX functions, sorted alphabetically. This list includes functions that KaTeX supports and some that it doesn't support. There is a similar page, with functions [sorted by type](./supported.md).
+
+## Symbols
+
+|Symbol/Function |  Rendered   | Source or Comment|
+|:---------------|:------------|:-----------------|
+|\!|$n!$|`n!`|
+|#|$\def\bar#1{#1^2} \bar{y}$|`\def\bar#1{#1^2} \bar{y}`|
+|\\#|$\#$||
+|%||`%this is a comment`|
+|\\%|$\%$||
+|&|$\begin{matrix} a & b\cr c & d \end{matrix}$|`\begin{matrix}`<br>&nbsp;&nbsp;&nbsp;`a & b \\`<br>&nbsp;&nbsp;&nbsp;`c & d`<br>`\end{matrix}`|
+|\\&|$\&$||
+|'|$'$||
+|`\'`|$\text{\'{a}}$|`\text{\'{a}}`|
+|(|$($||
+|)|$)$||
+|\\ |$a\ b$|`a\ b`|
+|\\"|$\text{\"{a}}$|`\text{\"{a}}`|
+|\\$|$\text{\textdollar}$||
+|\\,|$a\,\,{b}$|`a\,\,{b}`|
+|\\.|$\text{\.{a}}$|`\text{\.{a}}`|
+|\\:|$a\:\:{b}$|`a\:\:{b}`|
+|\\;|$a\;\;{b}$|a`\;\;{b}`|
+|_|$x_i$|`x_i`|
+|\\_|$\_$||
+|\\\`|$\text{\`{a}}$|<code>\text{\'{a}}</code>|
+|&#060;|$<$||
+|\\=|$\text{\={a}}$|<code>\text{\={a}}</code>|
+| >|$>$||
+|\[|$[$||
+|\]|$]$||
+|{|${a}$|`{a}`|
+|}|${a}$|`{a}`|
+|\\{|$\{$|`\{`|
+|\\}|$\}$|`\}`|
+|&#124;|$\vert$||
+|~|$\text{no~no~no~breaks}$|`\text{no~no~no~breaks}`|
+|\\~|$\text{\~{a}}$|<code>\text{\~{a}}</code>|
+|\\\\|$\begin{matrix} a & b\\ c & d\end{matrix}$|`\begin{matrix}`<br>&nbsp;&nbsp;&nbsp;`a & b \\`<br>&nbsp;&nbsp;&nbsp;`c & d`<br>`\end{matrix}`|
+|^|$x^i$|`x^i`|
+|\\^|$\text{\^{a}}$|<code>\text{\^{a}}<\code>|
```

Comment:
Backslash not visible in the source (needs double escaping)
Backslash instead of slash is used in the closing tag